app/src/main/java/com/gdg/android/MainActivity.kt

```diff
@@ -0,0 +1,129 @@
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Divider
+import androidx.compose.material3.DividerDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Color.Companion.LightGray
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.compose.AsyncImagePainter
+import coil.compose.rememberImagePainter
+import com.gdg.android.ui.theme.GDGAndroidTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            MainScreen()
+        }
+    }
+}
+
+@Composable
+fun MainScreen() {
+    val hobbies = listOf(
+        "독서", "영화 감상", "음악 감상", "산책", "뜨개질", "기타 연주"
+    )
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+    ) {
+        ProfileHeader(
+            imageUrl = "https://avatars.githubusercontent.com/u/140608054?v=4",
+            name = "이현진",
+            status = "숙명여자대학교 인공지능공학부 23학번입니다."
+        )
+
+        Spacer(modifier = Modifier.height(24.dp))
+
+        Text(
+            text = "취미",
+            style = MaterialTheme.typography.headlineMedium.copy(fontWeight = FontWeight.Bold),
+            modifier = Modifier.padding(bottom = 8.dp)
+        )
+
+        LazyColumn {
+            items(hobbies) { hobby: String ->
+                Text(
+                    modifier = Modifier.padding(vertical = 16.dp),
+                    text = hobby
+                )
+                HorizontalDivider(
+                    modifier = Modifier.fillMaxWidth(),
+                    color = LightGray,
+                    thickness = 1.dp
+                )
+
+            }
+        }
+    }
+}
+
+@Composable
+fun ProfileHeader(imageUrl: String, name: String, status: String) {
+    Row(
+        modifier = Modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        AsyncImage(
+            model = imageUrl,
+            contentDescription = "Profile Image",
+            modifier = Modifier
+                .clip(CircleShape)
+                .size(130.dp)
+                .padding(8.dp),
```

Comment:
Modifier는 적용 순서가 매우 중요합니다. 지금 CircleShape으로 모양을 자른 후 패딩을 적용하고 있기에 원본 사이즈가 줄어들며 왜곡이 발생하고 있습니다. 
원형을 유지하면서 패딩을 추가하려면 padding()을 clip()보다 앞에 두어 패딩을 먼저 적용한 후, CircleShape을 적용하면 됩니다.
```suggestion
            modifier = Modifier
                .padding(8.dp)
                .clip(CircleShape)
                .size(130.dp),
```

Reply:
그래서였군요..! 감사합니다😊